skyvern/llm/providers/ollama.py

```diff
@@ -0,0 +1,34 @@
+import os
+import requests
+from skyvern.llm.base import BaseLLMProvider
+
+class OllamaProvider(BaseLLMProvider):
+    """
+    Provider for Ollama's OpenAI-compatible local server.
+    """
+
+    def __init__(self, server_url: str = "http://localhost:11434", model: str = "deepseek-coder:6.7b"):
```

Comment:
I would remove the default here WDYT?
```suggestion
    def __init__(self, server_url: str = "http://localhost:11434", model: str = "deepseek-coder:6.7b"):
```

Reply:
yep removing would be better, and it's done